components/comment/demo/list.md

```diff
@@ -0,0 +1,66 @@
+---
+order: 1
+title:
+  zh-CN: 用法用名单
+  en-US: Usage with list
+---
+
+## zh-CN
+
+基本评论使用`antd` List组件来呈现一系列注释。
+
+## en-US
+
+A basic comment used with `antd` List component to render a series of comments.
+
+````jsx
+import { Comment, Tooltip, List } from 'antd';
+import moment from 'moment';
+
+const data = [
+  {
+    actions: [<span>Reply to</span>],
+    author: 'Han Solo',
+    avatar: 'https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png',
+    content: (
+      <p>We supply a series of design principles, practical patterns and high quality design resources (Sketch and Axure), to help people create their product prototypes beautifully and efficiently.</p>
+    ),
+    datetime: (
+      <Tooltip title={moment().subtract(1, 'days').format('YYYY-MM-DD HH:mm:ss')}>
+        <span>{moment().subtract(1, 'days').fromNow()}</span>
+      </Tooltip>
+    ),
+  },
+  {
+    actions: [<span>Reply to</span>],
+    author: 'Han Solo',
+    avatar: 'https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png',
+    content: (
+      <p>We supply a series of design principles, practical patterns and high quality design resources (Sketch and Axure), to help people create their product prototypes beautifully and efficiently.</p>
+    ),
+    datetime: (
+      <Tooltip title={moment().subtract(2, 'days').format('YYYY-MM-DD HH:mm:ss')}>
+        <span>{moment().subtract(2, 'days').fromNow()}</span>
+      </Tooltip>
+    ),
```

Comment:
We could make `Tooltip` and `moment.fromNow()` as built-in feature, the datetime could accept `string` or `Date` instance.

Reply:
Additionly, datetime was been used to be a hash link to comment item commonly.

<img width="798" alt="image" src="https://user-images.githubusercontent.com/507615/47891494-6c63c400-de8e-11e8-944a-a79d2bcfdaba.png">

We should consider this situation too.

Reply:
If the user does not want to show `Tooltip`? he would have to pass some additional prop, currently he can use `datetime` to fully handle that node, regarding hash I would suggest:

```
datetime={(
  <a href="#comment-8456748">
    <Tooltip title={moment().format('YYYY-MM-DD HH:mm:ss')}>
      <span>{moment().fromNow()}</span>
    </Tooltip>
  </a>
)}
```